doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3706,6 +3706,21 @@ NCSDK-6238: Socket API calls might hang when using Download client
 
   **Workaround:** Set :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_RANGE_REQUESTS` with the :ref:`lib_download_client` library.
 
+.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+
+Download client might fail with error ``-EMSGSIZE`` when using TLS on nRF91 Series devices.
```

Comment:
```suggestion
Download client might fail with error ``-EMSGSIZE`` when using TLS on nRF91 Series devices
```

Reply:
We would need a Jira ID for this Known issue.

Reply:
The issue could be moved to the "Libraries for networking" section as this lib belongs there - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/known_issues.html#libraries_for_networking

Reply:
Moved.